order/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from product.models import Product
+from django.conf import settings
+
+class Order(models.Model):
+    product = models.ForeignKey(
+        Product,
+        on_delete=models.CASCADE
+    )
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+    PAID = 'order is paid'
+    NOT_PAID = 'order is not paid'
```

Comment:
It is something we are going to store in the database. Let's use short strings with underscores
```suggestion
    PAID = 'paid'
    NOT_PAID = 'not_paid'
```

Reply:
done